app/lib/account/backend.dart

```diff
@@ -235,6 +241,7 @@ class AccountBackend {
       p.likes--;
       tx.queueMutations(inserts: [p], deletes: [likeKey]);
     });
+    await cache.userPackageLikes(user.userId).purge();
```

Comment:
Nit: Ideally we also would like to remove the like and purge the related cache if a package gets deleted. However, if we want to retain the deleted `Package` object, that may not be necessary and we can keep the entity and also the cache. Not sure if this needs any action now.

Reply:
It's on my todo implement removal of likes when we delete a package or a user. I'll create some issues.